client-ts/Microsoft.AspNetCore.SignalR.Client.TS/HubConnection.ts

```diff
@@ -51,6 +59,11 @@ export class HubConnection {
     }
 
     private processIncomingData(data: any) {
+        // For some reason, our TypeScript refs are set up for clearTimeout to take NodeJS.Timer... which is weird.
```

Comment:
Actually, I figured this out and it's intentional. This time around we're building our JS client as a NodeJS library and using Browserify to polyfill, rather than as a browser library, so we do want to use the NodeJS timer pattern.